guest-os-info.md

```diff
@@ -0,0 +1,34 @@
+## Guest Operating System Information
+
+Guest operating system identity for the Virtual Machine will be provided by the label ``kubevirt.io/os-short-id`` :
+
+```
+metadata:
+  name: myvm
+  labels:
+    kubevirt.io/os-short-id: win2k12r2
+```
+
+The ``kubevirt.io/os-short-id`` label is based on the short OS identifier from [libosinfo](https://libosinfo.org/)
+database.
+The following Short IDs are currently supported:
+
+| Short ID | Name | Version | Family | ID |
+| --- | --- | --- | --- | --- |
+| **fedora26** | Fedora 26 | 26 | linux | http://fedoraproject.org/fedora/26 |
+| **fedora27** | Fedora 27 | 27 | linux | http://fedoraproject.org/fedora/27 |
+| **rhel7.0** | Red Hat Enterprise Linux 7.0 | 7.0 | linux | http://redhat.com/rhel/7.0 |
+| **rhel7.1** | Red Hat Enterprise Linux 7.1 | 7.1 | linux | http://redhat.com/rhel/7.1 |
+| **rhel7.2** | Red Hat Enterprise Linux 7.2 | 7.2 | linux | http://redhat.com/rhel/7.2 |
+| **rhel7.3** | Red Hat Enterprise Linux 7.3 | 7.3 | linux | http://redhat.com/rhel/7.3 |
+| **rhel7.4** | Red Hat Enterprise Linux 7.4 | 7.4 | linux | http://redhat.com/rhel/7.4 |
+| **win2k12r2** | Microsoft Windows Server 2012 R2 | 6.3 | winnt | http://microsoft.com/win/2k12r2 |
+| **win2k16** | Microsoft Windows Server 2016 | 1709 | winnt | http://microsoft.com/win/2k16 |
+
+
+### Use with presets
+
+A Virtual Machine Preset representing an operating system with a ``kubevirt.io/os-short-id`` label could be applied on any given
+Virtual Machine that have and match the``kubevirt.io/os-short-id`` label.
```

Comment:
pls squash this line into the previous

Reply:
Could you add a link from this section to the VM preset guide (if exists) ?

Reply:
>Could you add a link from this section to the VM preset guide (if exists) ?

once the presets user guide will be merged